packages/service/db/migrations/20210722181845_update-created-at.sql

```diff
@@ -0,0 +1,9 @@
+-- migrate:up
+ALTER TABLE document_updates_queue ADD created_at timestamp DEFAULT now() NOT NULL
+ALTER TABLE document_updates_queue drop update_time;
+
+-- migrate:down
+ALTER TABLE document_updates_queue ADD update_time timestamp NOT NULL;
+ALTER TABLE document_updates_queue DROP created_at;
```

Comment:
I think the DROP should be DROP COLUMN <name>

Reply:
> The key word COLUMN is noise and can be omitted.

https://www.postgresql.org/docs/12/sql-altertable.html